### PR TITLE
Updated guide cards, covid page for covid launch

### PR DESCRIPTION
**Pull request by JacobDFrank**
- Increased accesibilty for the cards by adding more visible buttons and a border
- Made the banner and font sizes on the covid page smaller on mobile
- removed the intercom logo on mobile
- added padding to the google doc embed

### CODE REVIEW

File: app/pages/Covid/Covid.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Footer from '../../components/ui/Footer/Footer';
+import Iframe from 'react-iframe';
```
Comment (AnnaKalkanis) [inline app/pages/Covid/Covid.jsx:3]: I'm not familiar with this library. Is there a reason we need it here? Was the regular iframe not working?

File: app/pages/Covid/Covid.jsx [hunk 2]
```diff
@@ -15,30 +16,19 @@ export default class Covid extends React.Component {
...
-              of resources for individuals experiencing homelessness. We will be updating the list continuously, when services change, to ensure you have the most up to date information.
-              Please email
-              {' '}
-              <a href="mailto:contact@sheltertech.org">contact@sheltertech.org</a>
-              {' '}
-              to suggest edits.
-              See what types of places are open or closed at
-              {' '}
-              <a href="sf.gov/stay-home-except-essential-needs">sf.gov/stay-home-except-essential-needs</a>
+              This list is compiled to link individuals experiencing homelessness to resources.  We will be updating the list continuously, when services change. Please email <a href="contact@sheltertech.org">contact@sheltertech.org</a> to suggest edits.
```
Comment (AnnaKalkanis) [inline app/pages/Covid/Covid.jsx:22]: Might be worth throwing a `mailto:` on here so this is recognized as an email address. ie `href="mailto:contact@sheltertech.org"`

File: app/pages/Covid/Covid.jsx [hunk 3]
```diff
@@ -15,30 +16,19 @@ export default class Covid extends React.Component {
...
             </p>
           </header>
-          <section>
-            <iframe
+          <section className={styles.aboutSection}>
```
Comment (AnnaKalkanis) [inline app/pages/Covid/Covid.jsx:25]: don't think we need this class here - I'm not seeing it defined in the css

**Comment by AnnaKalkanis** [comment]
Definitely looks much better when resized! And I like the borders on the cards. The biggest thing for me is that it would be nice to avoid adding the react-iframe dependency if we don't need it.

**Comment by AnnaKalkanis** [comment]
You've also got some failing tests. Looks like just linting. You can see what they are by clicking the `details` for `CodeFactor` and `continuous-integration/travis-ci/pr `

File: app/pages/Covid/Covid.jsx (hunk 1 above)
Comment (JacobDFrank) [reply app/pages/Covid/Covid.jsx:3]: It allows us to easily strip out the CSS of a google chrome embed and allow it to be more readable. There may be some better workarounds but I wanted to get this out sooner.

File: app/pages/Covid/Covid.jsx (hunk 2 above)
Comment (JacobDFrank) [reply app/pages/Covid/Covid.jsx:22]: good idea, i'll add that

File: app/pages/Covid/Covid.jsx (hunk 3 above)
Comment (JacobDFrank) [reply app/pages/Covid/Covid.jsx:25]: you're right, I accidentally left that in

Comment (JacobDFrank) [reply app/pages/Covid/Covid.jsx:25]: I accidentaly added it.

File: app/pages/Covid/Covid.jsx (hunk 1 above)
Comment (AnnaKalkanis) [reply app/pages/Covid/Covid.jsx:3]: ah ok cool, sounds good!

**Pull request merged**